apps/playground-web/components/Playground/Playground.tsx

```diff
@@ -55,6 +58,22 @@ function Header() {
         />
         <h2 className="font-light text-2xl ml-2">PlayGround</h2>
       </div>
+        <Link
+          href="https://github.com/DiceDB/dice/issues/new/choose"
```

Comment:
Thanks for the PR @MehulKChaudhari. 
This `submit issue` button is taking us to the main dicedb repo. Since this is displayed on playground it would make more sense to redirect to https://github.com/DiceDB/playground-mono/issues/new for playground related bugs. 
cc @lucifercr07

Reply:
Sure

Reply:
But the issue can be related to the UI?

Reply:
Hey Guys @ayushsatyam146 @lucifercr07. 

please confirm this, so i can quicky make the change :)

Reply:
I think for now you can just send it to playground-mono repo only since UI is a shared repo for playground and other  desktop UI components as well.

Reply:
ok, 

Thank you.